File: rb/test/exists_test.rb
# UvIndexApi2 SDK exists test

require "minitest/autorun"
require_relative "../UvIndexApi2_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = UvIndexApi2SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- UvIndexApi2 SDK exists test

local sdk = require("uv-index-api2_sdk")

describe("UvIndexApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
